bin/node/runtime/src/lib.rs

```diff
@@ -607,7 +617,7 @@ construct_runtime!(
 		Indices: pallet_indices::{Module, Call, Storage, Config<T>, Event<T>},
 		Balances: pallet_balances::{Module, Call, Storage, Config<T>, Event<T>},
 		TransactionPayment: pallet_transaction_payment::{Module, Storage},
-		Staking: pallet_staking::{Module, Call, Config<T>, Storage, Event<T>},
+		Staking: pallet_staking::{Module, Call, Config<T>, Storage, Event<T>, ValidateUnsigned},
```

Comment:
I could actually take a first step toward this deprecation and implement this with a signed extension? Staking already has one signed extension that is added to the list.

Reply:
(not a big deal)